dataeng/resources/setup-platform-venv-py3.sh

```diff
@@ -3,12 +3,16 @@
 # This version of setup-platform-venv is specifically designed for newer
 # python3 versions of the platform.
 
+
+# Create and activate a virtualenv in shell script
+VENV="venv-${BUILD_NUMBER}"
+virtualenv --python=python3.8 --clear "${VENV}"
+source "${VENV}/bin/activate"
+
 # Install requirements
 pushd edx-platform
 # This is the same pip version we currently pin in our devstack edxapp container:
-pip install pip==20.0.2
+pip install pip==20.2.3
 make requirements
 popd
 
-# Save virtualenv location
-echo "PLATFORM_VENV=${VIRTUAL_ENV}" > platform_venv
```

Comment:
This will be excluded right ? @HassanJaveed84

Reply:
We still need to as the worker jobs need to know the location of platform's virtualenv. See comment above.

Reply:
I expect you'll still need this:
```
echo "PLATFORM_VENV=${VENV}" > platform_venv
```
the file `platform_venv` is used in downstream jobs, e.g. https://github.com/edx/jenkins-job-dsl/blob/7dda40199dcf01170a471ba552fc8cd2816297b1/dataeng/resources/setup-exporter.sh#L40